---
title: Galaxy Trainer Directory
---
<div class="panel panel-default">
  <div class="panel-body">
    <div class="embed-responsive embed-responsive-16by9">
      <iframe class="embed-responsive-item" src="https://www.google.com/maps/d/embed?mid=1r8LJy6la-JeIrg23aZjpwVNjJDE"></iframe>
    </div>
  </div>
</div>

The *Galaxy Trainer Directory* is a service of the [Galaxy Training Network](/src/teach/gtn/index.md) which aims to make it easy to share and find [Galaxy training resources](/src/teach/resources/index.md).  This directory lists individuals and groups that do Galaxy training.  The contacts listed here are willing to present Galaxy at meetings and workshops.  The directory includes particular areas of expertise and how far the instructors are willing to travel.

<div class="alert alert-success" role="alert">Do you or your organization do Galaxy training? Then please tell the Galaxy Community! You can either edit this wiki page directly (you will need to create a login), or **submit the information through [this online form.](http://bit.ly/gxytrainerform)**
</div>

# Trainers

Trainers are grouped by their *reach*.  That is, by how far they are willing to travel and who they are eligible to teach.  
* **[Global](/src/teach/trainers/index.md#global):** around the world
* **[Continental](/src/teach/trainers/index.md#continental):** across their continent 
* **[Regional](/src/teach/trainers/index.md#regional):** Willing to travel beyond their local area, but not across their continent
* **[Local](/src/teach/trainers/index.md#local):** These trainers don't travel, but their workshops are open to researchers outside their institution.
* **[Institution](/src/teach/trainers/index.md#institution):** Trainers that can only teach to researchers that are affiliated with their institution.

## Global

Trainers are potentially willing to travel globally to present Galaxy Training.

### The Centre for Bioinformatics and Computational Biology

<div class='right'><a href='http://www.up.ac.za/centre-for-bioinformatics-and-computational-biology'><img src="/src/images/logos/UPretoriaWide.png" alt="The The Centre for Bioinformatics and Computational Biology, University of Pretoria" width="210" /></a></div>

[The The Centre for Bioinformatics and Computational Biology's](http://www.up.ac.za/centre-for-bioinformatics-and-computational-biology) function is training, support and research in the areas of Bioinformatics and Computational Biology. We function as part of [The Genomics Research Institute](http://www.up.ac.za/the-genomics-research-institute) and [The African Centre for Gene Technologies](http://www.acgt.co.za/).

**Domains:**

  * General usage 
  * Analysing and comparing protein and nucleic acid sequences
  * Annotating genomes
  * Gene expression patterns
  * Investigating molecular structures
  * Simulating metabolic pathways

**Reach:** Most training is in southern Africa, but available globally. Located in Pretoria, South Africa.

**Links & Contact:**

  * [Contact](http://www.up.ac.za/en/centre-for-bioinformatics-and-computational-biology/article/29861/contact-us)
  * [The Centre for Bioinformatics and Computational Biology, University of Pretoria](http://www.up.ac.za/centre-for-bioinformatics-and-computational-biology)

### The FMI Bioinformatics Core Facility

<div class='right'><a href='http://www.fmi.ch'><img src="/src/images/logos/FMILogo.jpg" alt="The FMI Bioinformatics Core Facility" width="180" /></a></div>

We mainly provide internal training for our local Galaxy server. However, we are also involved in training and consulting outside our institute.

**Domains:**

  * General usage 
  * NGS analysis 

**Reach:** Primarily central Europe, but available globally.  Located in Basel, Switzerland.

**Links & Contact:**

  * [FMI Bioinfomatics platform](http://www.fmi.ch/bioinformatics)   
  * Email contact: [Hans-Rudolf Hotz](mailto:hrh AT fmi DOT ch)

### Freiburger Galaxy Project

<div class='right'><img src="/src/images/logos/bioinf-fr-logo-blau.jpg" alt="Galaxy Project" width="100" /></div>

The Freiburger Galaxy Project is led by Rolf Backofen from the University of Freiburg, Germany. The project is offerring Galaxy support and training for the University of Freiburg and beyond.  

**Domains:**

  * General usage 
  * Chip-Seq 
  * RNA-Seq 
  * Variant detection
  * Chemoinformatics
  * Genome annotation
  * Defining Tools in Galaxy
  * Installation and Administration

**Reach:** based in Freiburg (Germany), but used globally

**Links & Contact:**

  * [Bjoern Gruening](/src/people/bjoern-gruening/index.md)
  * [galaxy AT informatik DOT uni-freiburg DOT org](mailto:galaxy AT informatik DOT uni-freiburg DOT org) 

### Galaxy Project

<div class='right'><img src="/src/images/galaxy-logos/galaxy_project_logo_blue.png" alt="Galaxy Project" width="150" /></div>

The Galaxy Project does training at [events and institutions](/src/events/index.md) around the world.  These cover the whole range of Galaxy related topics.  Please contact Galaxy Outreach if you are interested in hosting training.

**Domains:**

  * General usage; several specific types of analysis
  * Galaxy on the cloud
  * Tool Development for Galaxy, including tool wrapping practices and the Galaxy ToolShed
  * Installation and Administration

**Reach:** Global, [Project members](/src/galaxy-team/index.md) are located in Pennsylvania, Maryland, Virginia, Georgia, California, and Oregon in the United States, and in Melbourne, Australia.

**Links & Contact:**

  * [Galaxy Team](/src/galaxy-team/index.md)
  * [Galaxy Outreach](/src/outreach/index.md), [outreach AT galaxyproject DOT org](mailto:outreach AT galaxyproject DOT org) 

### GalaxyP Project

<div class='right'><a href='http://usegalaxyp.org/'><img src="/src/images/logos/GalaxyPLogo.png" alt="GalaxyP Project" width="100" /></a></div>

Our training is part of the [GalaxyP Project](http://usegalaxyp.org/). We have training modules planned for using Galaxy for proteomics. This includes using [ProteinPilot](http://z.umn.edu/ppingp) (licensed software) within Galaxy on our local server. We plan to develop tutorials for use of [PeptideShaker](http://www.uib.no/en/rg/probe/65218/peptideshaker), proteogenomics analysis and metaproteomics analysis.

**Domains:** Proteomics tools within Galaxy.

**Reach:** Part of the University of Minnesota. Travel for Conferences, potentially around the world.

**Links & Contact:**

  * [usegalaxyp.org](http://usegalaxyp.org/) - the project's public web server
  * [Webinar on using ProteinPilot for database search](http://z.umn.edu/ppingp) within Galaxy-P.
  * [Pratik Jagtap](mailto:pjagtap AT umn DOT edu)
  * [Timothy Griffin, PI](mailto:tgriffin  AT umn DOT edu)

### Geneclat Technologies

<div class='right'><a href='http://geneclat.com/'><img src="/src/images/logos/GenEclatLogo.png" alt="Geneclat Technologies" width="160" /></a></div>

We at [Geneclat](http://geneclat.com/) provide Training for individual or group of people who are interested in NGS and Bioinformatics

**Domains:** General usage.

**Reach:** Based in Bengaluru, Karnataka, India; but available globally (and ready to travel!)

**Links & Contact:** [GenEclat Technologies](http://geneclat.com/)

### Memorial Sloan Kettering Cancer Center, Rätsch Laboratory

<div class='right'><a href='http://oqtans.org/'><img src="/src/images/logos/QqtansLogoWText.png" alt="Memorial Sloan Kettering Cancer Center, Rätsch Laboratory" width="170" /></a></div>
 
We have efficient computational methods in modeling all processes involved in RNA expression, regulation and processing. Most of the trainings are based on Oqtans package.

**Domains:** Deep Sequencing Data Analysis.

**Reach:** Based in New York City, but willing to train around the world.

**Links & Contact:**

  * [Oqtans home page](http://oqtans.org/)
  * [Oqtans Galaxy server](https://galaxy.cbio.mskcc.org/)

### MMG@IICT

<div class='right'><a href='http://mpds.osdd.net/'><img src="/src/use/mpdstb/mpdstb.png" alt="MMG@IICT" width="170" /></a></div>
 
Workshops and training about the Molecular Property Diagnostic Suite (MPDS).

**Domains:** CADD

**Reach:** Based in Hyderabad, India, but willing to train around the world.

**Links & Contact:**

  * [MPDS OSDD home page](http://mpds.osdd.net/)
  * [Contacts](http://mpds.osdd.net/galaxy/static/contact.htm)

### Noor Biotechnologies Ltd.

<div class='right'><a href='http://noorbiotech.com/'><img src="/src/images/logos/NoorBioTechLogo.jpg" alt="Noor Biotechnologies Ltd." width="270" /></a></div>

In collaboration with a number of research scientists with hands on experience in genomics and bioinformatics, [Noor Biotechnologies](http://noorbiotech.com/) offer consultancy services and onsite practical training in NGS and Bioinformatics including:

  * Library Preparation for DNA Sequencing on Illumina MiSeq and HiSeq
  * Illumina Library QC and Troubleshooting 
  * Optimizing Cluster Densities on Illumina MiSeq and HiSeq
  * Next Generation DNA Sequencing Technologies and Data Management
  * Basic Concepts in Bioinformatics and Making Sense of Data
  * Using Galaxy for Analysis of High Throughput Sequence Data

**Domains:**

  * *De novo* assembly
  * Small genome annotation
  * Mapping and variant calling
  * Multiple sequences alignment
  * Phylogeny
  * Multi Locus Sequencing Typing
  * Microbial species identification (16S and 18S metagenomics)
  * Antibiotics resistance predictions

**Reach:** Based in Malaysia and Saudi Arabia. We are available to train around the world.

**Links & Contact:**

  * [Contact information](http://noorbiotech.com/index.php/contact-us/)
  * [Email](mailto:info@noorbiotech.com)

### OpenHelix

<div class='right'><a href='http://openhelix.com/'><img src="/src/images/logos/OpenHelixLogo.jpg" alt="OpenHelix" width="150" /></a></div>

[OpenHelix](http://openhelix.com/) specializes in end-user training on a wide range of bioinformatics databases and software tools. Our goal is to reach new users with introductory information that can get them started using tools that are new to them, or to build their confidence by improving their grasp of the foundational features of the tools. We offer workshops or online training.

**Domains:**  General usage

**Reach:** It is easiest for us to travel in North America, but we have also delivered training in Europe, Asia, South America, and Africa. We can also do webinar format training. We are limited to delivering training in English, however.

**Links & Contact:**

  * [OpenHelix](http://www.openhelix.com/)
  * [Contact](http://www.openhelix.com/cgi/contact.cgi)
  * [OpenHelix online Intro to Galaxy Tutorial](http://www.openhelix.com/galaxy) (requires a [subscription](http://www.openhelix.com/cgi/subscriptions.cgi))

### QCIF

<div class='right'><a href='http://www.qcif.edu.au/'><img src="/src/images/logos/QCIFlogo.jpg" alt="QCIF" /></a></div>
 
 [QCIF](http://www.qcif.edu.au/) offers a range of Galaxy-based workshops though their bioinformatics division [QFAB Bioinformatics](http://qfab.org/training/). Workshops are run in partnership with [Australian BioCommons](https://www.biocommons.org.au/) and [Galaxy Australia](https://usegalaxy.org.au)

**Domains:**

  * General usage
  * RNA-Seq 
  * Variant detection 
  * Bacterial de novo genome assembly

**Reach:** Primarily Queensland, Australia, but available globally

**Links & Contact:**

  * [QCIF website](https://www.qcif.edu.au)
  * [QFAB Bioinformatics website](https://qfab.org)
  * E-mail contact [training AT qcif DOT edu DOT au](mailto:training AT qcif DOT edu DOT au)

### Stingray@Galaxy

<div class='right'><a href='http://www.fiocruz.br'><img src="/src/images/logos/FIOCRUZLogo.png" alt="Stingray@Galaxy"  /></a></div>
 
 [Stingray@Galaxy](http://galaxy.biowebdb.org/) was developed by the Lab of Computational and Systems Biology (LBCS) of the [Oswaldo Cruz Institute, FIOCRUZ](http://www.ioc.fiocruz.br/). Rodrigo Jardim and Alberto Dávila (LBCS) offer training at Fiocruz (and more recently researchers based on universities of Rio de Janeiro state) and beyond on the usage of Stingray@Galaxy.

**Domains:**

  * NGS data manipulation 
  * Genome assembly 
  * Comparative genomics

* *Reach:** We are based at Rio de Janeiro (Brazil) but available globally.

**Links & Contact:**

  * [BiowebDB Consortium](http://www.biowebdb.org) 
  * [Analysis and annotation course (2014)](http://systemsbiology.biowebdb.org/?p=1546)
  * [NGS data manipulation course (2014)](http://systemsbiology.biowebdb.org/?p=1449) 
  * [Stingray@Galaxy](http://galaxy.biowebdb.org)
  * Contacts: [Rodrigo Jardim](mailto:jardim AT ioc DOT fiocruz DOT br), [Alberto Dávila](mailto:davila AT fiocruz DOT br) 

### University of Bradford, School of Medical Sciences, Krzysztof Poterlowicz

<div class='right'><a href='https://scholar.google.co.uk/citations?user=46KP3twAAAAJ&hl=en'><img src="/src/teach/trainers/UBradfordPoterlowicz.png" alt="Krzysztof Poterlowicz, University of Bradford" width="250" /></a></div>
 
 [Krzysztof Poterlowicz](https://scholar.google.co.uk/citations?user=46KP3twAAAAJ&hl=en) of [The School of Medical Sciences](http://www.bradford.ac.uk/life-sciences/medical-sciences/) of the [University of Bradford](http://www.bradford.ac.uk/) offers a wide range of  Galaxy-based trainings including (Assembly, RNAseq, ChIPseq, Hi-C, DNase-Seq, Pathway enrichment)

**Domains:**

  * General usage
  * Assembly
  * RNA-Seq
  * ChIP-Seq
  * Hi-C
  * DNase-Seq,
  * Ontology and  pathway enrichment

**Reach:** Available Globally

**Links & Contact:**

  * [Krzysztof Poterlowicz on Google Scholar](https://scholar.google.co.uk/citations?user=46KP3twAAAAJ&hl=en)
  * [Contact info](http://www.bradford.ac.uk/life-sciences/medical-sciences/our-staff/dr-krzysztof-poterlowicz.php)

### Victorian Life Sciences Computation Initiative (VLSCI)

<div class='right'><a href='http://www.vlsci.org.au/'><img src="/src/images/logos/VLSCILogo.png" alt="Victorian Life Sciences Computation Initiative (VLSCI)" width="150" /></a></div>
 
[VLSCI](http://www.vlsci.org.au/) is a petascale facility incorporating the Life Sciences Computation Centre, led by Andrew Lonie and staffed with over 20 bioinformaticians. Amongst a suite of training courses on offer is a series of Galaxy-based bioinformatics workshops, each taught by an expert in that particular domain e.g. cancer genomics, microbial genomics etc. The materials are designed for both self-directed learning and as content for delivering workshops.

**Domains:**

  * General usage 
  * Chip-Seq 
  * RNA-Seq 
  * Variant detection 
  * Bacterial de novo genome assembly 
  * 16S Metagenomics

**[GTN Training Resources](/src/teach/resources/index.md):**

  * [RNA-Seq Differential Gene Expression Introductory Tutorial](/src/teach/resource/gvl-rna-seq-tutorial/index.md)
  * [RNA-Seq Differential Gene Expression Advanced Tutorial](/src/teach/resource/gvl-rna-seq-advanced-tutorial/index.md)
  * [Variant Detection Introductory Tutorial](/src/teach/resource/gvl-variant-detection-introductory-tutorial/index.md)
  * [Variant Detection Introductory Tutorial](/src/teach/resource/gvl-variant-detection-advanced-tutorial/index.md)
  * [de novo Genome Assembly (Small Genome)](/src/teach/resource/gvl-de-novo-genome-assembly/index.md)

**Reach:** Australian-based, but used globally

**Links & Contact:**

  * [VLSCI Website](http://vlsci.org.au)
  * [GVL Tutorials](http://genome.edu.au/wiki/Learn)
  * [Bio page for Andrew Lonie](http://vlsci.org.au/researcher/alonie), Head, VLSCI's Life Sciences Computation Centre
  * Email [Andrew Lonie](mailto:alonie @ unimelb DOT edu DOT au) for further enquiries or to invite LSCC staff to present at your conference, workplace.

## Continental

Trainers that are potentially willing to travel, roughly within their home continent.

### CRS4 - Centre for Advanced Studies, Research and Development in Sardinia

<div class='right'><a href='http://www.crs4.it/'><img src="/src/images/logos/CRS4Logo.jpg" alt="CRS4" width="200" /></a></div>

* *Description:*
    The CRS4's Bioinformatics Laboratory offers bioinformatics training and project-based data analysis. The training is based on  [Orione](http://orione.crs4.it/), our Galaxy-based platform for NGS analysis. CRS4 is at the forefront in the research, development and application of technology for acquisition, processing, distribution, and visualization of massive data (Big Data) with a special emphasis on bio-medical and healthcare applications.

**Domains:**
  * General usage 
  * Variant detection
  * RNA-Seq
  * Metagenomics
  * Microbiology
  * Data visualization
  * Installation and administration
  * Tool Development

**Reach:**
  * Regional. Trainings are offered primarily in Italy, but available in Europe.

**Links & Contact:**
  * [CRS4 Bioinformatics main website](http://www.bioinformatica.crs4.it/)
  * [CRS4 Bioinformatics Training Program site](http://www.bioinformatica.crs4.it/training/)
  * Contact: [Giorgio Fotia](mailto:ngs-courses AT crs4 DOT it)

### EMBL-ABR

<div class='right'><a href='https://www.embl-abr.org.au/training/'><img src="/src/images/logos/EMBL-ABR-logo.jpg" alt="EMBL-ABR" width="200" /></a></div>

[EMBL Australian Bioinformatics Resource](https://www.embl-abr.org.au/) is a distributed national research infrastructure providing bioinformatics support to life science researchers in Australia. EMBL ABR aims to provide new opportunities and harmonise existing training efforts across Australia to maximise the reach and scope of bioinformatics training, as well as to share experience and best practice from Australia with international initiatives. The [EMBL ABR Training programme](https://www.embl-abr.org.au/training/) comprises a series of focal topics to tackle training gaps with the aim of enabling sustainable and scalable bioinformatics use by life scientists across Australia. There are three main factors contributing to training gaps among life science researchers: 

 1. insufficient exposure to the principles and application of bioinformatics; 
 1. the speed at which technologies, data analysis practices and tools develop and the lag in how these are incorporated into the current undergraduate and postgraduate curricula 
 1. a lack of support within some research groups or institutions for students and postdoctoral researchers to learn bioinformatics skills and tools. 

**Domains:**

  * Currently, in terms of topics we offer training on: 
  * Introduction to Galaxy 
  * Galaxy Workflows
  * Using Histories and Workflows

**Reach:** Primarily Australia, but we welcome opportunities to contribute when invited to training efforts world wide.

**Links & Contact:**

  * [EMBL-ABRF Training Programme](https://www.embl-abr.org.au/training/)
  * [EMBL-ABR home](https://www.embl-abr.org.au/)
  * [About us](https://www.embl-abr.org.au/about/)
  * Contact: [Vicky Schneider](mailto:mv DOT schneider AT unimelb.edu.au)

### GenoToul bioinformatics platform, Sigenae, NED (GenPhySE) and TWB

<div class='right'><a href='http://bioinfo.genotoul.fr/index.php?id=10'><img src="/src/images/logos/GenoToulLogo.png" alt="GenoToul Training" width="200" /></a><br />
<a href='http://www.sigenae.org/'><img src="/src/images/logos/SigenaeLogo.png" alt="Sigenae" width="200" /></a>
</div>

The [GenoToul bioinformatics platform](http://bioinfo.genotoul.fr/), [Sigenae](http://www.sigenae.org/), NED ([GenPhy](/src/GenPhy/index.md)SE) and TWB offers a [catalog of training sessions](http://bioinfo.genotoul.fr/index.php?id=10) on Galaxy. Team members include Sarah Maman, Philippe Bardou, Sabrina Rodriguez, Celine Noirot, Cédric Cabau, Géraldine Pascal, Maria Bernard, Frederic Escudié, Laurent Cauquil, and Mahendra Mariadassou.

**Domains:**

  * Galaxy : First steps (1 day) 
  * Galaxy: Reads alignment and SNP calling (1,5 days) 
  * Galaxy : RNAseq alignment and transcripts assemblies (1.5 days) 
  * Metagenomics: amplicon analysis (4 days)

**Reach:** Based in Toulouse, France; potentially willing to present across Europe

**Links & Contact:**

  * [GenoToul Training](http://bioinfo.genotoul.fr/index.php?id=10)
  * [GenoToul bioinformatics facility](http://bioinfo.genotoul.fr/)

## Regional

*Regional* is roughly defined as "willing to travel beyond my local area, but not across the continent."

### Alberta Children's Hospital Research Institute

<div class='right'><a href='http://research4kids.ucalgary.ca/'><img src="/src/images/logos/ACHRILogo300.jpg" alt="Alberta Children's Hospital Research Institute" width="180" /></a></div>

[ACHRI](http://research4kids.ucalgary.ca/) has three research foci, with one being The Molecular and Genetic Basis of Child Health. With $5M in funding from the Alberta Children's Hospital Foundation, next-generation sequencing and bioinformatics platform was established in 2011 to advance personalized medicine and basic science research at the [University of Calgary](http://ucalgary.ca/). Bioinformatics efforts have largely centered around new tools and workflows for optimizing exome genotyping, quality control and variant triage based on clinical symptoms.

**Domains:**

  * Exome and whole genome variant analysis
  * RNASeq
  * General usage
  * Developing tool wrappers

**Reach:** Have taught for the local Calgary community only so far, but would be willing to do roadshows globally on exome analysis using Galaxy and our custom tools as we roll out a public version of the portal.

**Links & Contact:**

  * [Alberta Children's Hospital Research Institute](http://research4kids.ucalgary.ca/)
  * Contact: [Paul Gordon](mailto:gordonp AT ucalgary DOT ca)

### Bioinformatics.ca

<div class='right'><a href='http://bioinformatics.ca/workshops/'><img src="/src/teach/trainers/BioinfCaHT_seq_image_0.png" alt="Bioinformatics.ca"  /></a></div>

[Bioinformatics.ca](http://bioinformatics.ca/workshops/) through its Canadian Bioinformatics Workshops (CBW) series began offering one and two week short courses in bioinformatics, genomics and proteomics in 1999. In the fall of 2007, bioinformatics.ca moved to its new host at the Ontario Institute for Cancer Research and the CBW initiated development of a new format and series of exciting workshops. It was recognized that with the introduction of new technologies and scientific approaches to research, having the computational biology capacity and skill to deal with this new data has become an even greater asset. This new series of workshops focuses on training the researchers of these advanced technologies on the latest approaches being used in computational biology to deal with the new data. In one of the workshops we teach, [Informatics on High Throughput Sequencing Data](http://bioinformatics.ca/workshops/2013/informatics-high-throughput-sequencing-data), we teach 1/2 day on use of Galaxy in NGS analysis. We cover an overview of Galaxy as well as an example of RNA-Seq and SNP-calling (lab).

**Domains:**

  * Alignment (lab) 
  * SNP-calling (lab)
  * RNA-Seq (lecture)

**Reach:** Courses are in Toronto (Canada), and are open to all world-wide.

**Links & Contact:**

  * [Bioinformatics.ca Workshops](http://bioinformatics.ca/workshops/)
  * [Contact](https://bioinformatics.ca/contact-us)

### Abdelazeem Abdelhameed Elhabyan

Abdelazeem Abdelhameed Elhabyan is a medical doctor in Egypt, and an online master student of biomedical diagnostics Arizona State University.

**Domains:**  

  * Introduction to Galaxy project
  * Main Features of Galaxy
  * NGS and manipulation
  * RNA-seq
  * ChIP -seq

**Reach:** Arabic-speaking region and Africa

**Links & Contact:**

  * [Email](mailto:Abdelazeem_Abdelhameed2015505@yahoo.com)


### Galaxy @ PRABI Project

<div class='right'><img src="/src/images/logos/PRABILogo.png" alt="Galaxy@prabi Project" width="150" /></div>

The galaxy@prabi project is led by Vincent Navratil from the [University of Lyon](http://www.univ-lyon1.fr/), France. The project is offerring Galaxy service, support and training for and with research teams working in the perimeter of the [PRABI](http://www.prabi.fr) core facility (Rhône Alpes Region, mainly Lyon and Grenoble) but also beyond the Alps montains.  

**Domains:**  

  * General usage 
  * RNA-Seq, Differential Expression, Variant calling and functional impact prediction
  * PRABI tools ([kissplice](http://kissplice.prabi.fr), kissDE, discoSNP, [priam](http://priam.prabi.fr))
  * Defining Tools in Galaxy
  * Installation and Administration

**Reach:** Rhône Alpes regions (France), beyond the Alps montains if requested.

**Links & Contact:**

  * galaxy.prabi.fr and toolshed.prabi.fr soon available in production
  * galaxy@prabi user list [galaxy-user AT listes DOT univ-lyon1 DOT fr](mailto:galaxy-user AT listes DOT univ-lyon1 DOT fr) (in french)
  * galaxy@prabi support [galaxy-support AT listes DOT univ-lyon1 DOT fr](mailto:galaxy-support AT listes DOT univ-lyon1 DOT fr) 

### Galaxy User Group Grand Ouest (GUGGO)

<div class='right'><a href='https://www.e-biogenouest.org/groups/guggo'><img src="/src/teach/trainers/eBiogenouest6blanc.png" alt="e-Biogenouest GUGGO Initiative" width="150" /></a></div>
 
[GUGGO](https://www.e-biogenouest.org/groups/guggo) is a working group gathering 6 Western France Life Sciences Computation Centre. From West to East: 

  * [Cellule Bioinformatique-Ifremer](http://wwz.ifremer.fr/pcim) Ifremer/U.B.O./CNRS/ (Brest/Plouzane, contact: [Laure Quintric](mailto:bioinfo-contact AT ifremer DOT fr))
  * [ABiMS IFB core facility](http://abims.sb-roscoff.fr/training) UPMC/CNRS (Roscoff, contact: [Christophe Caron](mailto:christophe DOT caron AT fr))
  * [GenOuest IFB core facility](http://www.genouest.org/) Rennes1 University/CNRS/INRIA (Rennes, contact: [Yvan Le Bras](mailto:yvan.le_bras AT irisa DOT fr))
  * [BIPAA](http://www6.inra.fr/bipaa) INRA (Rennes, contact: [Anthony Bretaudeau](mailto:anthony DOT bretaudeau AT irisa DOT fr))
  * [BIRD IFB core facility](http://www.pf-bird.univ-nantes.fr/) Nantes University/INSERM (Nantes, contact: [Audrey Bihouée](mailto:audrey DOT bihouee AT univ-nantes DOT fr))
  * [IRHS BioInformatics team](https://www6.angers-nantes.inra.fr/irhs/Recherche/BioInfo) Angers University/Agrocampus Ouest/INRA (Angers, [Jean-Pierre Renou](mailto:jean-pierre DOT renou AT angers DOT inra DOT fr)).  

**Domains:**

  * Life science
    * General usage
    * DNA-seq, RNA-seq, RAD-Seq
    * Without assembly NGS data treatment
    * Metagenomics
    * Transcriptomics
    * Phylogeny
  * Bioinformatics
    * Galaxy tools development, integration and sharing
    * Galaxy on the Cloud

**Reach:** Primarily (western) France but available globally

**Links & Contact:**

  * [GUGGO Website](https://www.e-biogenouest.org/groups/guggo)
  * [Training sessions](https://www.e-biogenouest.org/einfrastructure/education)
  * Email [Yvan Le Bras](mailto:yvan.le_bras AT irisa DOT fr) for further informations.

### LUMC, ErasmusMC, DTL Learning Programme

<div class='right'><div class='center'>
<a href='https://www.lumc.nl/'><img src="/src/images/logos/LUMCLogo.png" alt="LUMC" width="200" /></a><br /><br />
<a href='http://www.erasmusmc.nl/'><img src="/src/images/logos/ErasmusMCLogo.png" alt="ErasmusMC" width="150" /></a><br /><br />
<a href='http://www.dtls.nl/about/programmes/dtl-learning/'><img src="/src/images/logos/DTLLogo.png" alt="Dutch Techcentre for Life Science Learning Programme" width="155" /></a>
</div></div>
 
Bioinformatics groups at LUMC and ErasmusMC contributed to several courses on using Galaxy for NGS data analysis. We also plan to sustain those courses on a yearly base. The organization of these courses is supported by the [DTL learning program](http://www.dtls.nl/about/programmes/dtl-learning/).

**Domains:**  
  * General usage
  * RNA-seq
  * Assembly
  * Variant calling and filtering
  * Building tools in Galaxy
  * Interaction with other workflow systems

**Reach:** We organize and contribute to courses within the Netherlands.

**Links & Contact:**

  * [DTL Learning Programme](http://www.dtls.nl/about/programmes/dtl-learning/)
  * Contacts: [Leon Mei](mailto:h DOT mei AT lumc DOT nl), [Andrew Stubbs](mailto:a DOT stubbs AT erasmusmc DOT nl) 

### Ruđer Bošković Institute (RBI)

<div class='right'><div class='center'>
<a href='http://www.irb.hr/eng'><img src="/src/images/logos/IRBLogo300.jpg" alt="Ruđer Bošković Institute (RBI)" width="135" /></a><br /><br />
</div></div>

The [Ruđer Bošković Institute](http://www.irb.hr/eng) is Croatia’s leading scientific institute in the natural and biomedical sciences as well as marine and environmental research.

**Domains:**  

  * General usage
  * Galaxy on the Cloud; CloudMan
  * Installation and Administration
  * RNA-Seq

**Reach:** We organize and contribute to training within Croatia.

**Links & Contact:**

  * [Ruđer Bošković Institute (RBI)](http://www.irb.hr/eng)
  * Contacts: [Davor Davidović](http://www.irb.hr/eng/People/Davor-Davidovic), [Centre for Informatics and Computing](http://www.irb.hr/eng/Research/Divisions-and-Centers/Centre-for-Informatics-and-Computing)
  
### Systems Biology and Bioinformatics (SBI)

<div class='right'><div class='center'>
<a href='https://sbi.uni-rostock.de'><img src="/src/images/logos/SBILogo.png" alt="Systems Biology and Bioinformatics (SBI)" width="135" /></a><br /><br />
</div></div>

The Department of [Systems Biology and Bioinformatics](https://sbi.uni-rostock.de) focuses on the development of systems- and control methodologies, using mathematical modelling, statistical data analysis, and computational approaches applied to complex dynamical systems.

**Domains:**  

  * General usage
  * Quality control and data preprocessing
  * RNA-Seq analysis
  * Mapping
  * Visualization
  
**Reach:** We organize and contribute to training within Germany.

**Links & Contact:**

  * [Systems Biology and Bioinformatics (SBI)](https://sbi.uni-rostock.de)
  * Contacts: [Olaf Wolkenhauer](https://www.sbi.uni-rostock.de/team/single/prof-olaf-wolkenhauer/)


### South Green Platform

<div class='right'><a href='http://www.southgreen.fr/'><img src="/src/images/logos/SouthGreenBig.png" alt="South Green Platform" width="160" /></a></div>
 
[South Green](http://www.southgreen.fr/) is a bioinformatic platform, studying tropical crops. It uses Galaxy for a large panel of bioinformatic analyses, from RNA-seq to phylogeny and 3D protein structure prediction.

**Domains:**  

  * RNA-seq
  * assembly
  * mapping
  * SNP calling
  * annotation
  * phylogeny
  * 3D protein structure

**Reach:** We teach mainly in Montpellier (France), but also anywhere in France in collaboration with the IFB (Bioinformatic French Institut).

**Links & Contact:**

  * [South Green Platform](http://www.southgreen.fr/)
  * [Contact](http://www.southgreen.fr/contact)

## Local

These trainers don't travel, but their workshops are open to researchers outside their institution.

### Earlham Institute

<div class='right'><a href='http://earlham.ac.uk/scientific-training'><img src="/src/images/logos/EarlhamInstituteBig.png" alt="Earlham Institute" width="180" /></a></div>
 
The [Earlham Institute](http://earlham.ac.uk/)'s [Scientific Training](http://earlham.ac.uk/scientific-training) team work with content experts from EI and across the world to create hands-on training courses and workshops in cutting edge genomics, bioinformatics and high-performance computing.

**Domains:** General usage and many specialisations

**Reach:** Workshops are held in Norwich, United Kingdom, and are open to all researchers and students.

**Links & Contact:**

  * [EI Scientific Training](http://earlham.ac.uk/scientific-training)
  * E-mail contact [training AT earlham DOT ac DOT uk](mailto:training AT earlham DOT ac DOT uk)

### Peter Mac

<div class='right'><a href='https://www.petermac.org/'><img src="/src/images/logos/PeterMac.png" alt="Peter MacCallum Cancer Center" width="200" /></a></div>
 
[Peter Mac](https://www.petermac.org/) is one of the world’s leading cancer research, education and treatment centres globally and is Australia’s only public hospital solely dedicated to caring for people affected by cancer. We have over 2,500 staff, including more than 580 laboratory and clinical researchers, all focused on providing better treatments, better care and potential cures for cancer.

**Domains:** General usage, cancer genomics

**Reach:** Training is held at [Peter Mac](https://www.petermac.org/) in Melbourne, Australia, and are open to all researchers and students.

**Links & Contact:**

  * [Peter Mac](https://www.petermac.org/)
  * E-mail contact [Maria Doyle](mailto:Maria DOT Doyle AT petermac DOT org)

### Texas A&M University/Center for Phage Technology

<div class='right'><a href='https://cpt.tamu.edu/'><img src="/src/images/logos/TAMULogo.png" alt="Texas A&M University/Center for Phage Technology" width="150" /></a></div>

The TAMU Galaxy offers access a local Galaxy instance for TAMU Staff, faculty, and researchers. The CPT@TAMU offers a Galaxy instance catering to Bacteriophage Research. 

**Domains:**  

  * General usage 
  * Variant detection
  * Genome/Transcriptome Assembly
  * Genome annotation

**Reach:** Based in College Station, Texas. 

**Links & Contact:**

  * [TAMU Galaxy](http://galaxy.tamu.edu)
  * [CPT Public Galaxy](http://cpt.tamu.edu/galaxy-pub/)

### UC Davis Genome Center's Bioinformatics Core Facility

<div class='right'><a href='http://bioinformatics.ucdavis.edu/'><img src="/src/images/logos/UCDavisGenomeCenter_Bioinf.png" alt="UC Davis Genome Center's Bioinformatics Core Facility" width="200" /></a></div>
 
The UCD Bioinformatics Core offers Galaxy-based training using the Amazon Cloud, as well as command line training and project-based consulting and data analysis.

**Domains:**

  * General usage
  * Alignment
  * Variant Calling
  * Genome (DNA) Assembly
  * RNA-Seq
  * ChIP-Seq
  * miRNA-Seq

**[GTN Training Resources](/src/teach/resources/index.md):**

  * [Galaxy Training AMI](/src/teach/resource/uc-davis-core-ami/index.md)
  * [Using Galaxy for Analysis of High Throughput Sequence Data Workshop](/src/teach/resource/uc-davis-core-galaxy-course/index.md)

**Reach:** We teach on campus in Davis, California, USA - workshops are open to the public

**Links & Contact:**

  * [UCD Bioinformatics Core main website](http://bioinformatics.ucdavis.edu/)
  * [Bioinformatics Training Program site](http://training.bioinformatics.ucdavis.edu/)
  * [UCD Genome Center Training site](http://training.genomecenter.ucdavis.edu/) 
  * [UC Davis Bioinformatics Training Program FAQ](https://training.bioinformatics.ucdavis.edu/faq/)
  * Contact us [via email](mailto:bioinformatics DOT core AT ucdavis DOT edu)

### University of Cambridge Bioinformatics Training Facility

<div class='right'><a href='http://www.bio.cam.ac.uk/facilities/research/training/bioinformatics'><img src="/src/images/logos/UniCambridgeLogoWide.png" alt="University of Cambridge Bioinformatics Training Facility" width="200" /></a></div>
 
The [Bioinformatics Training Facility](http://www.bio.cam.ac.uk/facilities/research/training/bioinformatics) of the School of Biological Sciences of the University of Cambridge offers a wide range of bioinformatics training courses, including Galaxy-based training.

**Domains:**

  * General usage 
  * Alignment 
  * Variant Calling 
  * RNA-Seq
  * ChIP-Seq

**Reach:** Workshops take place at the University of Cambridge, and are open to people outside the University of Cambridge.

**Links & Contact:**

  * [Bioinformatics training at the University of Cambridge](http://training.csx.cam.ac.uk/bioinformatics/event-timetable)
  * Bioinformatics training manager: [Gabriella Rustici](mailto:gr231 AT cam DOT ac DOT uk) 

### University of Minnesota Minnesota Supercomputing Institute Research Informatics Solutions

<div class='right'><a href='https://www.msi.umn.edu/about/staff/riss'><img src="/src/images/logos/UofM_D2D.gif" alt="University of Minnesota Minnesota Supercomputing Institute Research Informatics Solutions" width="300" /></a></div>
 
[RIS](https://www.msi.umn.edu/about/staff/riss) is a group of experienced bioinformatics experts within the Supercomputing Institute at the University of Minnesota.

**Domains:**
  * General usage 
  * RNA-Seq
  * ChIP-Seq

**Reach:** Workshops take place at the University of Minnesota, and are open to people outside the University of Minnesota.

**Links & Contact:**

  * Access [all MSI tutorials](https://msi.umn.edu/tutorials)
  * Contact [Lauren Mills](mailto:ljmills AT umn DOT edu) for more info.

## Institution

Trainers that can only teach to researchers that are affiliated with their institution.


### HSPH Bioinformatics Core (HBC)

<div class='right'><a href='https://hsphbio.ghost.io/'><img src="/src/images/logos/HBC.HSPH.logo.png" alt="HSPH (Harvard School of Public Health) Bioinformatics Core" width="220" /></a></div>

[HBC](https://hsphbio.ghost.io/)'s staff are available to help with the management, integration, and contextual analysis of high-throughput biological data, focusing on next-generation sequencing support. We provide consulting, training, tools, databases and best practices, while fostering collaboration and a community of bioinformatic activities. We also build unified infrastructure to support a diverse set of experimental systems and high-throughput biological data.

**Domains:**  

  * General Introduction to Galaxy
  * RNA-Seq 
  * ChIP-Seq 
  * Exome-Seq

**[GTN Training Resources](/src/teach/resources/index.md):**

  * [ChIP-Seq Workshop](/src/teach/resource/chb-harvard-chip-seq/index.md)
  * [Running your own Galaxy instance (in the cloud)](/src/teach/resource/chb-harvard-cloudman/index.md)
  * [Exome-Seq Workshop](/src/teach/resource/chb-harvard-exome-seq/index.md)
  * [Introduction to Galaxy](/src/teach/resource/chb-harvard-chip-seq/index.md)
  * [RNA-Seq Workshop](/src/teach/resource/chb-harvard-rna-seq/index.md)

**Reach:** Limited to Harvard researchers with specific affiliations.

**Links & Contact:**

  * [HSPH Bioinformatics Core (HBC)](https://hsphbio.ghost.io/)
  * [Radhika Khetani](mailto:rkhetani AT hsph DOT harvard DOT edu)


# Frequently Asked Questions

**Q: Have instructors in this directory been vetted in any way?**

No, not in any rigorous or official way.  However, trainers listed here are well-known in their communities.
<br /><br />

**Q: What does it cost to bring a trainer to my institution or event?**

That depends entirely on who does the training.  Most/All of these groups are **not** specifically funded to do Galaxy Training outside their home organization.  It may require creative funding and/or a nearby trainer to get Galaxy Training at your institution or event.  The trainer location map can help you identify nearby trainers.

**Q: Where can I find other non-Galaxy specific bioinformatics training?**

<div class='right'><a href='http://mygoblet.org'><img src="/src/images/logos/GobletLogo.png" alt="Global Organisation for Bioinformatics Learning, Education & Training (GOBLET)" height="140" /></a></div>

Most of the members of the GTN also do bioinformatics training that isn't Galaxy-based.  If you want training that involves both Galaxy-based and non-Galaxy-based material than the GTN membership is the best place to start.  For general bioinformatics training a great place to look for trainers is the [Global Organisation for Bioinformatics Learning, Education & Training (GOBLET)](http://mygoblet.org). GOBLET was founded in 2012 by a group of international & national societies & networks to offer an umbrella for bioinformatics, biocuration, biocomputing & computational biology learning, education & training. It has online resource directories, very similar to what the GTN has.

And, as of 2015 the [Galaxy Training Network](/src/teach/gtn/index.md) is a proud member of [GOBLET](http://mygoblet.org).
